script/os-autoinst-setup-multi-machine

```diff
@@ -45,11 +45,15 @@ install_packages() {
 
 configure_firewall() {
     systemctl enable --now firewalld
+    default_zone=$(firewall-cmd --get-default-zone)
     firewall-cmd --permanent --get-services | grep -q isotovideo && firewall-cmd --permanent --delete-service=isotovideo
     firewall-cmd --permanent --new-service isotovideo
     for i in $(seq 1 "$instances"); do firewall-cmd --permanent --service=isotovideo --add-port=$((i * 10 + 20003))/tcp; done
     firewall-cmd --permanent --zone="$zone" --add-service=isotovideo
-    firewall-cmd --get-default-zone | grep -q "$zone" || firewall-cmd --set-default-zone="$zone"
+   if  [[ $default_zone != "$zone" ]]; then
+     firewall-cmd -q --permanent --remove-interface="$ethernet" --zone="$default_zone"
+     firewall-cmd --set-default-zone="$zone"
+   fi
```

Comment:
Now it looks like the indentation is wrong and there's a duplicate whitespace:
```suggestion
    if [[ $default_zone != "$zone" ]]; then
      firewall-cmd -q --permanent --remove-interface="$ethernet" --zone="$default_zone"
      firewall-cmd --set-default-zone="$zone"
    fi
```

Reply:
I just fixed this in the latest commit, let's see